### PR TITLE
Update ome-advanced-message-encryption.md

### PR DESCRIPTION
**Pull request by lindabrGithub**
@KCCross we have request from EEE to add the following note due to PG escalation.  Can you help with this.

"Advanced Message Encryption revocation and expiration functions will not function unless the "Premium Encryption in Office 365" option is enabled in their E5 license."

### CODE REVIEW

File: microsoft-365/compliance/ome-advanced-message-encryption.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ Advanced Message Encryption helps customers meet compliance obligations that req
 
 You can only revoke and set an expiration date for emails sent to external recipients.
 
+Note:"Advanced Message Encryption revocation and expiration functions will not function unless the "Premium Encryption in Office 365" option is enabled in their E5 license."
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-advanced-message-encryption.md]:
```suggestion
> [!NOTE]
> The Advanced Message Encryption revocation and expiration functions will not work unless you enable the **Premium Encryption in Office 365** option in your E5 license.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lindabrGithub, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by yogkumgit** [comment]
@lindabrGithub Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@lindabrGithub Gentle reminder

**Comment by lindabrGithub** [comment]
@yogkumgit , sorry for the delay, I missed the comment.  Let me know if there is anything else you need me to do to complete this.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Pull request merged**